docker-compose.yml

```
@@ -18,13 +18,13 @@ services:
       ExternalCatalogBaseUrl: http://localhost:5088
     ports:
       - "5088:80"
-    container_name: catalogapi
+    container_name: orderapi
```

This is the container name for the catalog service
```suggestion
    container_name: catalogapi
```